changelogs/unreleased/5933-investigate-timeout-on-license-component.yml

```
@@ -0,0 +1,4 @@
+description: Add get_latest_resources_resource_type_count to get the count for each resource_type over all resources in the model's latest version.
+issue-nr: 5933
+change-type: patch
```

This PR adds new functionality to inmanta-core, so it should be minor instead.

```suggestion
change-type: minor
```